centreon/tests/e2e/cypress/e2e/Dashboards/commands.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+
+import 'cypress-wait-until';
+
+Cypress.Commands.add('customWaitUntil', (accessRightsTestId) => {
```

Comment:
rename `customWaitUntil` with an appropriate name which explain the intention of the method
```suggestion
Cypress.Commands.add('customWaitUntil', (accessRightsTestId) => {
```

Reply:
I would rename the command `waitUntilForDashboardRoles`